zomes/blog/entry_types/post/validation/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "validation"
+version = "0.1.0"
+authors = ["Nicolas Luck <nicolas@lucksus.eu>"]
+
+[dependencies]
+serde = "1.0"
+serde_json = "1.0"
+serde_derive = "1.0"
+hdk = { git = "https://github.com/holochain/hdk-rust"}
```

Comment:
'branch'

Reply:
Just like I mentioned in my other PR, git refs assume master branch, so this actually works (since hdk-rust is on master default branch).